### DISCUSSION TITLE
[Open3D WARNING] GLFW Error: Cocoa: Failed to find service port for display

### DISCUSSION THREAD

**Discussion by MangroveGuy** [isl-org/Open3D#6702] [Q&A]
I am trying to display a point cloud using Open3d and getting a blank screen and the error message "**[Open3D WARNING] GLFW Error: Cocoa: Failed to find service port for display**". I have seen others complain about this, but no solutions work for me.

**Comment by Bridesicle** [comment]


**Reply by nicholasm112** [reply]
I haven't found any answers
